aeson-commit.cabal

```diff
@@ -1,56 +1,46 @@
-name:                aeson-commit
-version:             0.1.0.0
-synopsis:            Parse Aeson data with commitment
-description:         Aeson parsers backtracks too much for some use cases.
-                     The commit parser forbids backtracking for already committed parses.
-license:             BSD3
-author:              Cross Compass
-maintainer:          viktor.kronvall@cross-compass.com
-copyright:           2019 Cross Compass Ltd.
-category:            Web
-build-type:          Simple
-cabal-version:       >=1.10
-extra-source-files:  README.md
+cabal-version:      >=1.10
+name:               aeson-commit
+version:            1.0
+license:            BSD3
+copyright:          2019 Cross Compass Ltd.
+maintainer:         viktor.kronvall@cross-compass.com
```

Comment:
I think you should be a maintainer as well @jonascarpay.

Reply:
Can you have multiple maintainers?

Reply:
I made myself maintainer and we share authorship